From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martin Schmiedel <Martin.Schmiedel@tq-group.com>
Date: Tue, 27 May 2025 07:45:46 +0200
Subject: Revert "LFU-278-20 imx8m/imx93: Disable BINMAN for iMX8M/iMX93 EVK
 build"

This reverts commit a52f5af6be925fe14a282901e722543a0a6139cf.

The TQ-internal commit disabled BINMAN across all i.MX8M and i.MX9
configurations because certain NXP EVK board builds were broken by it.
In the Armbian build context only TQ SoM defconfigs are built; none of
the affected EVK configurations are used here.

BINMAN is required for correct flash.bin assembly on i.MX8MP targets
and is re-enabled here to unblock the following Makefile fix.
---
 arch/arm/mach-imx/imx8m/Kconfig | 1 +
 arch/arm/mach-imx/imx9/Kconfig  | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm/mach-imx/imx8m/Kconfig b/arch/arm/mach-imx/imx8m/Kconfig
index 111111111111..222222222222 100644
--- a/arch/arm/mach-imx/imx8m/Kconfig
+++ b/arch/arm/mach-imx/imx8m/Kconfig
@@ -2,6 +2,7 @@ if ARCH_IMX8M
 
 config IMX8M
 	bool
+	select BINMAN
 	select GICV3 if ARMV8_PSCI
 	select HAS_CAAM
 	select LTO
diff --git a/arch/arm/mach-imx/imx9/Kconfig b/arch/arm/mach-imx/imx9/Kconfig
index 111111111111..222222222222 100644
--- a/arch/arm/mach-imx/imx9/Kconfig
+++ b/arch/arm/mach-imx/imx9/Kconfig
@@ -7,6 +7,7 @@ config AHAB_BOOT
 
 config IMX9
 	bool
+	select BINMAN
 	select HAS_CAAM
 	select ROM_UNIFIED_SECTIONS
 
-- 
Armbian

